### ISSUE TITLE
[REQUEST] Some features

### ISSUE THREAD

**Issue by Uzurka** [swingmx/swingmusic#159]
Hello and firstly, thanks a lot for this new self-hosted opensource project which is exactly what i was looking for listen to my music ! 
I know this project is really new, but i wanted to submit some new features, without asking for an ETA or for them to be accomplished right now ! 
I listen to a lot of music, and i use to copy my physical copies on my homelab.
Here are some features i would love to see : 
- Android, iOS & AppleTV applications : So i can listen to my own music from everywhere !!
- Musicbrainz (or some other musicdb support) to automatically find & tag librairies
- Login system (as i will set it over the INTERNET with a reverse proxy, and i don't really want anyone to access & listen to my music)
- An equalizer, so i can perfectly manage how i want to listen to my music

Again, thanks for your work, i can't wait to see this project growing up <3

**Comment by mungai-njoroge** [comment]
Hey @Uzurka 

Thanks for the feature requests. For the musicdb support feature, I don't think that will be possible. I recommend you use software dedicated to that kind of work. Take a look at the [beets project](https://beets.io/).

Thanks.

**Comment by Uzurka** [comment]
No problem about that, the folder's navigation will be quite great for me too, that was just bonus ^^
You think the others are possible ? :)

**Comment by Gremious** [comment]
These should probably all be separate issues, but a simply user/password system would be very much appreciated.

**Comment by sortedcord** [comment]
I agree, if app analytics are going to be implemented (as mentioned in https://github.com/swing-opensource/swingmusic/issues/158#issuecomment-1811060864), a multi-user system is a must. This is such a cool project and I was instantly delighted when I saw there was support for artists separation and other small things that many other players don't have. I hope it sticks around, I would love to collaborate on it in the near future

**Comment by cwilvx** [comment]
Hello @sortedcord 

You mentioned you would like to collaborate on this project. I'm looking for collaborators on this project to make things go faster. If you're still interested, I'd love to give you an intro.

Feel free to drop me a message on Telegram: https://t.me/cwilvx
